src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-storage-class-form/rgw-storage-class-form.component.ts

```diff
@@ -230,128 +292,42 @@ export class RgwStorageClassFormComponent extends CdForm implements OnInit {
       this.onAllowReadThroughChange(value);
     });
   }
-  createForm() {
-    const self = this;
 
-    const lockDaysValidator = CdValidators.custom('lockDays', () => {
-      if (!self.storageClassForm || !self.storageClassForm.getRawValue()) {
-        return false;
-      }
-
-      const lockDays = Number(self.storageClassForm.getValue('read_through_restore_days'));
-      return !Number.isInteger(lockDays) || lockDays === 0;
-    });
-    this.storageClassForm = this.formBuilder.group({
-      storage_class: new FormControl('', {
-        validators: [Validators.required]
-      }),
-      zonegroup: new FormControl(this.selectedZoneGroup, {
-        validators: [Validators.required]
-      }),
-      region: new FormControl('', [
-        CdValidators.composeIf({ storageClassType: TIER_TYPE.CLOUD_TIER }, [Validators.required])
-      ]),
-      placement_target: new FormControl('', {
-        validators: [Validators.required]
-      }),
-      access_key: new FormControl(null, [
-        CdValidators.composeIf({ storageClassType: TIER_TYPE.CLOUD_TIER }, [Validators.required])
-      ]),
-      secret_key: new FormControl(null, [
-        CdValidators.composeIf({ storageClassType: TIER_TYPE.CLOUD_TIER }, [Validators.required])
-      ]),
-      target_path: new FormControl('', [
-        CdValidators.composeIf({ storageClassType: TIER_TYPE.CLOUD_TIER }, [Validators.required])
-      ]),
-      retain_head_object: new FormControl(true),
-      glacier_restore_tier_type: new FormControl(STORAGE_CLASS_CONSTANTS.DEFAULT_STORAGE_CLASS, [
-        CdValidators.composeIf({ storageClassType: TIER_TYPE.GLACIER }, [Validators.required])
-      ]),
-      target_endpoint: new FormControl('', [Validators.required, this.urlValidator]),
-      glacier_restore_days: new FormControl(STORAGE_CLASS_CONSTANTS.DEFAULT_GLACIER_RESTORE_DAYS, [
-        CdValidators.composeIf({ storageClassType: TIER_TYPE.GLACIER || TIER_TYPE.CLOUD_TIER }, [
-          CdValidators.number(false),
-          lockDaysValidator
-        ])
+  public createAcls(): CdFormGroup {
```

Comment:
This is only an ACL, right? So singular
```suggestion
  public createAcl(): CdFormGroup {
```